### ISSUE TITLE
Problem running with autosubmit 3.3.1 on ECMWF machine

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#120]
In GitLab by @Eleftheriaexarchou on May 12, 2016, 17:57

There is a problem when using the latest version of autosubmit 3.3.1 and trying to run in ecmwf: the error I get is     
```        
[CRITICAL] 2016-05-12 11:50:54,446 Unhandled exception on Autosubmit: Error sending file job via 'ecaccess-job-put': Usage:
    ecaccess-job-submit -version|-help|-manual
```      

you can see the log files here /esnas/autosubmit/a00p/tmp/20160512_115017_run.log for example. FYI that the ec-access module is loaded, and the ec-access certificate also works. People: @mmenegoz @ vguemas @dmanubens @ macosta

**Comment by kinow** [comment]
In GitLab by @Eleftheriaexarchou on May 12, 2016, 18:04

@ jlope2 ?

**Comment by kinow** [comment]
In GitLab by @dmanubens on May 17, 2016, 13:04

Status changed to closed by commit dfa5779b92c32546a823bf40472d2b8a713aa559
